miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/MiniAppSpec.kt

```diff
@@ -0,0 +1,28 @@
+package com.rakuten.tech.mobile.miniapp
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.Test
+import kotlin.reflect.KMutableProperty
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.jvm.isAccessible
+
+class MiniAppSpec {
+
+    @Test
+    fun `should update configuration when get instance of MiniApp`() {
+        val miniAppSdkConfig: MiniAppSdkConfig = mock()
+        val miniAppCompanion: MiniApp.Companion = MiniApp.Companion
+        val miniApp: MiniApp = mock()
+
+        val instance = MiniApp.Companion::class.memberProperties.first { it.name == "instance" }
```

Comment:
We typically avoid using reflection in tests. Is there another way to test this?

I guess you could make `instance` be `internal` with `@VisibleForTesting`? It's still not ideal, but it's better than using reflection.

Reply:
Unfortunately, I don't find another way to solve it so I applied your suggestion.